Sources/SmithyIdentityAPI/Context/Context+AuthSchemePreference.swift

```diff
@@ -16,6 +16,12 @@ extension Context {
     public func getAuthSchemePreference() -> [String]? {
         get(key: authSchemePreferenceKey)
     }
+
+    /// The auth scheme preferences for authenticating this request.
+    public var authSchemePreference: [String]? {
+        get { getAuthSchemePreference() }
+        set { set(key: authSchemePreferenceKey, value: newValue) }
+    }
```

Comment:
The Bedrock API Key customization needs to alter the configured auth scheme preference order, hence a setter is provided